client/src/components/classesPage/ClassCreateView.js

```diff
@@ -77,10 +75,132 @@ function ClassCreateView(props) {
     plain_content: campaignData.plain_content // requires [unsubscribe]
   };
 
+  useEffect(() => {
+    console.log(campaignData);
+  }, [campaignData]);
+
+  // sendgrix axios instance
+  const sgAx = axios.create({
+    baseURL: 'https://api.sendgrid.com/v3',
+    headers: {
+      authorization: `Bearer ${process.env.REACT_APP_SENDGRID_API_KEY}`
+    }
+  });
+
+  const token = localStorage.getItem('accessToken');
+
+  const ax = axios.create({
+    // baseURL: 'https://refreshr.herokuapp.com' // production
+    baseURL: 'http://localhost:9000',
+    headers: {
+      authorization: `Bearer ${token}` // development
+    }
+  });
+
+  const sgDb = async refreshrs => {
+    let firstPass = true; // flag to ensure we don't submit the same list to sendgrid
+    let list; // to store list id
+    for (let refreshr of refreshrs) {
+      console.log('list:', list);
+      console.log('refreshr:', refreshr);
+      // create list
+      console.log(listData.classnameInput);
+      console.log(recipientData);
+      if (firstPass) {
```

Comment:
Big if statement! Almost didn't see it.

A comment and space might help. :)

Reply:
Yes, good call!